### ISSUE TITLE
After install OZWP Hassio not start

### ISSUE THREAD

**Issue by ghost** [jkeljo/hassio-addons#8]
Hi
I installed OZWP and after install and reboot my hassio not boot. Home assistant web page not start.

**Comment by jkeljo** [comment]
Simply installing the plugin should have no effect on Hass itself; plugins are by design isolated from Hass itself. Starting the plugin will stop Hass, and you have to ssh in and start it again (instructions in the readme for this plugin), or if you can’t do that (don’t have the ssh plugin running, for example), rebooting the machine will do it in a pinch.

If none of that works, I’m going to need to see the home-assistant.log from a failed attempt to start Hass in order to troubleshoot further.

**Comment by ghost** [comment]
I reinstalled Hassio and again install plugin. ANd the same effect. After restart Home Assistant can not run.
Logs:
2019-10-15 21:09:15 WARNING (Dummy-2) [openzwave] Z-Wave Notification DriverFailed : {'notificationType': 'DriverFailed', 'homeId': 0, 'nodeId': 255}

**Comment by ghost** [comment]
Control panle OZWP works but Home Assisnat on port 8123 not working

**Comment by EtoTen** [comment]
I am getting the same behavior after installing this plugin on the latest Home Assistant 0.104.3 on a RPi. I can see the CP on http://hassio.local:8090/ but can no longer hit the hassio GUI @ http://hassio.local:8123/.

**Comment by jkeljo** [comment]
@EtoTen the behavior you describe is expected (and indeed documented in the usage instructions for this add-on). When the add-on starts OZWCP, it must stop HA since only one of them can access your Zwave controller at a time, and you must stop OZWCP and manually restart HA when you are done.

I *think* the issue you’ve commented on is reporting a different behavior, whereby merely installing the plugin (not starting it) prevents HA from booting successfully after a restart. I don’t know how that’s possible and haven’t been able to reproduce it myself.

**Comment by EtoTen** [comment]
> 
> 
> @EtoTen the behavior you describe is expected (and indeed documented in the usage instructions for this add-on). When the add-on starts OZWCP, it must stop HA since only one of them can access your Zwave controller at a time, and you must stop OZWCP and manually restart HA when you are done.
> 
> I _think_ the issue you’ve commented on is reporting a different behavior, whereby merely installing the plugin (not starting it) prevents HA from booting successfully after a restart. I don’t know how that’s possible and haven’t been able to reproduce it myself.
Yup, you are totally correct. My apologies, I assumed something was wrong when the hassio front end was no longer available and didn't realize that this was intended behavior.
